### PR TITLE
Upgrade to PyWPS 4.5.1

### LINKED ISSUE #160: Configure `storage_copy_function` to False
## Description

PyWPS 4.4.1 has new option to link output files instead of copying them. This has the potential to speed server response. 

- [ ] What are the implications in terms of data retention policies ?
- [ ] What is the performance improvement ?

### PR DESCRIPTION
**Pull request by huard**
## Overview

This PR will help resolve #160 because PyWPS 4.5.1 fixes an issue with recursive directory creation.

Changes:

* Upgrade PyWPS to 4.5.1
* Unpin birdy because conda package is mislabelled. 
* Bumpversion to 0.8.0

## Additional Information

Links to other issues or sources.

### CODE REVIEW

File: environment-docs.yml [hunk 1]
```diff
@@ -4,10 +4,10 @@ channels:
...
 dependencies:
-- pywps>=4.4.5
+- pywps >= 4.5.1
 - sphinx
 - nbsphinx
 - ipython
 - xclim~=0.31
 - matplotlib
-- birdy >= 0.8.1
+- birdy # >= 0.8.1 but pin fails because conda package seems to be tagged v0.8.1 instead of just 0.8.1.
```
Comment (Zeitsperre) [inline environment-docs.yml:13]: Oh, that's really odd! We need to fix that!

**Comment by huard** [comment]
@aulemahal I'd issue a release with this PR. Ok for you ?

**Comment by Zeitsperre** [comment]
Everything seems to be building fine. All good on my end!

**Comment by huard** [comment]
@tlvu FYI, this should help with #160 and https://github.com/bird-house/birdhouse-deploy/issues/11

**Review by Zeitsperre — APPROVED** [review]

**Comment by aulemahal** [comment]
Oh! Would you be ok with waiting tomorrow afternoon or monday? I want to do #208 before the next release...
It can also wait and be released as 0.8.1 if you need 0.8 quick.

**Comment by huard** [comment]
It's blocking a few things, so yeah, I'd like to release this ASAP and then we can do a 0.8.1 with xclim 0.32 and your new process.

**Comment by aulemahal** [comment]
Alright then!

**Pull request merged**